Update v3 token management docs and add management tokens (#5373)
influxdata/docs-v2: content/influxdb/clustered/reference/client-libraries/flight/python-flightsql-dbapi.md

Replace the following:

> - {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: an {{% product-name %}} [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the databases you want to query

Reviewer: ```suggestion
- {{% code-placeholder-key %}}`DATABASE_TOKEN`{{% /code-placeholder-key %}}: a [database token](/influxdb/clustered/admin/tokens/#database-tokens) with read permissions on the database you want to query
```
